sched/environ/env_unsetenv.c

```diff
@@ -61,23 +61,30 @@ int unsetenv(FAR const char *name)
 {
   FAR struct tcb_s *rtcb = this_task();
   FAR struct task_group_s *group = rtcb->group;
-  int ret = OK;
+  int idx;
 
-  DEBUGASSERT(name && group);
+  DEBUGASSERT(group);
+
+  /* Check the incoming parameter */
+
+  if (name == NULL || *name == '\0' || strchr (name, '=') != NULL)
```

Comment:
```suggestion
  if (name == NULL || name[0] == '\0' || strchr (name, '=') != NULL)
```
nit: just a suggestion, at least for me this way becomes more easily readable.

Reply:
At least setenv() uses the original style `*name == '\0'`also so I'll keep the original formatting